packages/couchbase/manifest.yml

```
@@ -80,7 +80,7 @@ vars:
     type: text
     title: Hosts for Sync Gateway prometheus exporter
```

When the host-config is optional at base level, then there is a possibility that user may enable the sync-gateway and fail to provide config inputs. To avoid this the config should be moved inside the policy template inputs section as suggested below and the required flag should be turned on. `required:true`

```suggestion
policy_templates:
  inputs:
    - type: prometheus/metrics
         title: Hosts for Sync Gateway prometheus exporter
```